### ISSUE TITLE
Bulk image conversion of DICOM files to a directory

### ISSUE THREAD

**Issue by Enet4** [Enet4/dicom-rs#343]
Currently the tool only supports reading and converting a single file at a time.

- The tool would need two new CLI options:
   - One to override the intended target image format through a separate option (e.g. `-f` or `-t`).
   - (`-D` or `--out-dir`) to specify the target directory to write all image files to.
- Then, it should be possible to pass one or more files and/or directories, like in `dicom-storescu`. When doing this, the two options above are required.

Example of use:

```none
mkdir -p outdir
dicom-toimage my/dicom/files/ -d outdir -t png
```

**Comment by abroy77** [comment]
hello, I'm looking for new good first issues to tackle. this one seems like it was addressed in #518. is that correct? Thank you!

**Comment by Enet4** [comment]
@abroy77 Right, thank you for bringing it up. There are a few other easy issues I can curate and expand.
